packages/backend/src/plugins/catalog.ts

```diff
@@ -45,7 +45,7 @@ export default async function createPlugin({
 
   useHotCleanup(
     module,
-    runPeriodically(() => higherOrderOperation.refreshAllLocations(), 10000),
+    runPeriodically(() => higherOrderOperation.refreshAllLocations(), 100000),
```

Comment:
of course this should be configurable (and perhaps with different numbers per input ...), but not today, and all I know is that 10 seconds is way too often to hammer APIs

Reply:
10s is pretty convenient for the components for local development, starting to wonder if we actually want varying refresh times somehow. We could do this backoff inside the processor at least for now, but gonna be a tricky bit to handle

Reply:
Let's leave it at this value for now, local dev does the update at startup and if one is in a hurry (this is a pretty uncommon thing to wait for i would expect), one can ctrl+c and start again :P

I'll make it configurable at a later time

Reply:
But mah local component filez

Fine :p